source/slang/slang-ir-inst-defs.h

```diff
@@ -1129,7 +1129,7 @@ INST(CudaKernelLaunch, CudaKernelLaunch, 6, 0)
 INST(GetEquivalentStructuredBuffer,     getEquivalentStructuredBuffer, 1, 0)
 
 /* Layout */
-    INST(VarLayout, varLayout, 1, HOISTABLE)
+    INST(VarLayout, varLayout, 1, 0)
```

Comment:
My understanding is that HOISTABLE means it is some kind of global property.
I think VarLayout should remain HOISTABLE.
What is the reason why you "don't believe it needs to be HOISTABLE."

Reply:
> "don't believe it needs to be HOISTABLE."

I checked and I realized my assumption was incorrect, we do need `HOISTABLE`. The reason we need it is due to the optional operands which need to be shared, this was an incorrect assumption (it is why tests are failing)